pyanaconda/modules/payload/live/live_image_interface.py

```diff
@@ -76,3 +103,70 @@ def VerifySSL(self) -> Bool:
     def SetVerifySSL(self, verify_ssl: Bool):
         """Set if the ssl verification should be enabled."""
         self.implementation.set_verifyssl(verify_ssl)
+
+    @property
+    def RequiredSpace(self) -> UInt64:
+        """Space required by the source image."""
+        return self.implementation.required_space
+
+    @emits_properties_changed
+    def SetRequiredSpace(self, required_space: UInt64):
+        """Set the space required by the source image."""
+        self.implementation.set_required_space(required_space)
+
+    def UpdateKernelVersionList(self):
+        """Update the list of kernel versions."""
+        self.implementation.update_kernel_version_list()
+
+    def GetKernelVersionList(self) -> List[Str]:
+        """Get the kernel versions list."""
+        return self.implementation.kernel_version_list
+
+    @dbus_signal
+    def KernelVersionListChanged(self, kernel_version_list: List[Str]):
+        """Signal kernel version list change."""
+        pass
+
+    def SetupWithTask(self) -> ObjPath:
+        """Setup the handler.
+
+        FIXME: use real name? CheckSourceImageWithTask
+        Check availability of the image and update required space
+        """
+        return self.implementation.check_source_image_with_task()
+
+    def PreInstallWithTask(self) -> ObjPath:
+        """Set up installation source image
+
+        FIXME: use real name? (SetupInstallationSourceImageTask)
+        * Download the image
+        * Check the checksum
+        * Mount the image
+        """
+        return self.implementation.setup_installation_source_image_with_task()
```

Comment:
Please, keep the names in the DBus API consistent with the names in the Python module. It improves the readability of the code.

Reply:
https://github.com/rhinstaller/anaconda/pull/2041/commits/54d7d8d5918bcb74d352327828d356f3891b7f4a